packages/grafana-ui/src/components/Icon/Icon.mdx

```diff
@@ -7,6 +7,14 @@ import { Icon } from './Icon';
 
 Grafana's wrapper component over [Font Awesome](https://fontawesome.com/) and Unicons icons
 
+## Accessibility
+
+`Icon` is aria-hidden by default. If your icon has semantic meaning, please provide a `title` or wrap it in a `Tooltip`.
```

Comment:
Actually, I do have one comment, I think we are better off recommending `aria-label` or `aria-labelledby` rather than `title`. In the screenshot below when it has a title the default browser tooltip appears covering some of the information in the Grafana tooltip.

![Screenshot of a Grafana tooltip being partially covered by a browser tooltip.](https://github.com/grafana/grafana/assets/6906380/10aae181-52ab-4e15-8b73-0f77e14bf308)

Reply:
I agree that this should be the case when using it with the `Tooltip`, but I think `title` is a better option when that's not the case, as it can provide information for non-screen readers too.